app/lib/account/backend.dart

```diff
@@ -121,7 +121,8 @@ class AccountBackend {
   /// not exists.
   ///
   /// Throws Exception if more then one `User` entry exists.
-  Future<User> lookupOrCreateUserByEmail(String email) async {
+  Future<User> lookupOrCreateUserByEmail(String email,
```

Comment:
How about creating two methods `lookupOrCreateUserByEmail` and `lookupUserByEmail`.

We can use `lookupUserByEmail` inside `lookupOrCreateUserByEmail`, so we don't have to do the lookup logic twice...

Reply:
PTAL, I've updated the code to use the two methods.